### PR TITLE
Support for Unicorn Chain

### PR DESCRIPTION
**Pull request by samedayjeremy**
Please consider adding Unicorn chain

### CODE REVIEW

**Comment by risasoft** [comment]
Hi mate... PR open for this https://github.com/chainapsis/keplr-chain-registry/pull/799

File: cosmos/unicorn-420.json
Comment (editaahn) [inline cosmos/unicorn-420.json:1]: Could you please update the name of this file to `unicorn` by removing the version?

Comment (samedayjeremy) [reply cosmos/unicorn-420.json:1]: yes!

Comment (samedayjeremy) [reply cosmos/unicorn-420.json:1]: all set @editaahn

**Comment by editaahn** [comment]
The image size should be 256 * 256px

**Pull request merged**